### PR TITLE
Alc 198 basic chart

### PR DESCRIPTION
**Pull request by ledes**
Please add an infura key to you env file after this is merged

### CODE REVIEW

File: .env.local.example [hunk 1]
```diff
@@ -1 +1,3 @@
 REACT_APP_INFURA_API_KEY=xxx
+NODE_PATH=src
+REACT_APP_NETWORK_ID=1
```
Comment (hosainnet) [inline .env.local.example:3]: You should copy this file as `.env.local` (which is already ignored by git) and add your secrets there

File: .gitignore [hunk 2]
```diff
@@ -67,3 +67,5 @@ yarn-error.log
 
 # direnv
 .envrc
+
+.env
```
Comment (hosainnet) [inline .gitignore:71]: .env should be checked in with defaults, and overrides go in `.env.local` as per the comment above

File: .env [hunk 3]
```diff
@@ -1,2 +0,0 @@
-NODE_PATH=src
-REACT_APP_NETWORK_ID=1
```
Comment (hosainnet) [inline .env:2]: Let's revert this please as with other comments

File: .env.local.example (hunk 1 above)
Comment (hosainnet) [reply .env.local.example:3]: See https://github.com/alchemistcoin/copper#recommended-setup

Comment (ledes) [reply .env.local.example:3]: 👍

Comment (hosainnet) [reply .env.local.example:3]: shall we revert this as they're defaulted in .env?

Comment (ledes) [reply .env.local.example:3]: yes, you are right

**Review by hosainnet — DISMISSED** [review]

**Review by hosainnet — APPROVED** [review]

**Pull request merged**